packages/file_selector/file_selector_windows/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:lints/recommended.yaml
```

Comment:
I think we can skip this file

Reply:
ok! I will remove this and the `ints` package